src/fstests/fstests.jinja2

```diff
@@ -135,30 +141,34 @@ class Job(BaseJob):
         return node
 
     def _run(self, src_path):
-        fail_results = {
+        results = {
             'node': {
                 'name': 'fstests',
-                'result': 'fail',
                 'state': 'done',
             },
             'child_nodes':[],
         }
```

Comment:
Why not use `_generate_node_result()` here?

Reply:
In this specific case, we have `state` in the root node, but I am considering this because the upcoming PR([this issue](https://github.com/orgs/kernelci/projects/15)) will insert a result in the root note so that we will have `fail` information in case of any failures during the fstests pipeline.